events/entities.lua

```diff
@@ -3,7 +3,11 @@ function on_post_entity_died(event)
     event_json["name"] = event.name
     event_json["event"] = "POST_DESTROYED"
     event_json["event_name"] = event.name
-    event_json["damage_type"] = event.damage_type.name
+    if event.damage_type then
+        event_json["damage_type"] = event.damage_type.name
+    else
+        event_json["damage_type"] = {}
```

Comment:
Could you update this to `event_json["damage_type"] = "no-damage"`? I thought I fixed this, but it looks like it didn't get added to my commit.

Basically anytime an entity is destroyed, it triggers this event, but in scenarios like, a cargo pod arriving at the destination, it is destroyed without damage.